Sources/StreamChat/Workers/Background/NewChannelQueryUpdater.swift

```diff
@@ -72,7 +72,14 @@ final class NewChannelQueryUpdater<ExtraData: ExtraDataTypes>: Worker {
         changes.forEach { change in
             switch change {
             case let .insert(channelDTO, _):
-                updateChannelListQuery(for: channelDTO)
+                database.write {
+                    let dto = $0.channel(cid: try! ChannelId(cid: channelDTO.cid))
+                    dto?.needsRefreshQueries = false
```

Comment:
I don't understand what is gooing on here 😅  Maybe add little documentation? This makes me wonder why it doesn't need to refresh queries...